source/index.js

```diff
@@ -123,15 +123,18 @@ module.exports = async (input = 'patch', options) => {
 			{
 				title: 'Installing dependencies using Yarn',
 				enabled: () => options.yarn === true,
-				task: () => exec('yarn', ['install', '--frozen-lockfile', '--production=false']).pipe(
-					catchError(error => {
-						if (error.stderr.startsWith('error Your lockfile needs to be updated')) {
-							return throwError(new Error('yarn.lock file is outdated. Run yarn, commit the updated lockfile and try again.'));
-						}
-
-						return throwError(error);
-					})
-				)
+				task: () => {
+					return exec('yarn', ['install', '--frozen-lockfile', '--production=false']).pipe(
+						catchError(async error => {
```

Comment:
Are you sure `catchError` can be async? Because if not, this is an unhandled rejection.

Reply:
Yes, i tested it again and it works async as we are piping the response.